### ISSUE TITLE
Conform.effective doesn't work for exrm (v0.19.6)

### ISSUE THREAD

**Issue by seantanly** [bitwalker/exrm-umbrella-test#3]
After upgrading to the latest exrm v0.19.6, I couldn't get conform to parse the configuration. I'm getting the following exception, which can be replicated in this umbrella test project, with the `mix.exs` updated to the latest exrm.

```
$ mix conform.new
==> The schema for your project has been placed in config/app_one.schema.exs
==> The schema for your project has been placed in config/app_two.schema.exs
==> The schema for your project has been placed in config/master_app.schema.exs
$ mix conform.configure
==> The .conf file for app_one has been placed in config/app_one.conf
==> The .conf file for app_two has been placed in config/app_two.conf
==> The .conf file for master_app has been placed in config/master_app.conf
$ mix conform.effective
** (Conform.Schema.SchemaError) Invalid schema. Should be a keyword list containing :mappings and :translations keys.
    lib/conform/translate.ex:169: Conform.Translate.to_config/3
    lib/mix/tasks/conform.effective.ex:84: Mix.Tasks.Conform.Effective.do_run/1
    (mix) lib/mix/project.ex:196: Mix.Project.in_project/4
    (elixir) lib/file.ex:1137: File.cd!/2
    lib/mix/tasks/conform.effective.ex:52: anonymous fn/4 in Mix.Tasks.Conform.Effective.run/1
    (elixir) lib/enum.ex:1387: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/mix/tasks/conform.effective.ex:51: Mix.Tasks.Conform.Effective.run/1
    (mix) lib/mix/cli.ex:55: Mix.CLI.run_task/2
```

Here's the `mix.lock`
```
%{"bbmustache": {:hex, :bbmustache, "1.0.3"},
  "conform": {:hex, :conform, "0.16.0"},
  "cowboy": {:hex, :cowboy, "1.0.3"},
  "cowlib": {:hex, :cowlib, "1.0.1"},
  "erlware_commons": {:hex, :erlware_commons, "0.15.0"},
  "exrm": {:hex, :exrm, "0.19.6"},
  "getopt": {:hex, :getopt, "0.8.2"},
  "neotoma": {:hex, :neotoma, "1.7.3"},
  "providers": {:hex, :providers, "1.4.1"},
  "ranch": {:hex, :ranch, "1.1.0"},
  "relx": {:hex, :relx, "3.5.0"}}
```

**Comment by seantanly** [comment]
The error seems to be fixed by confirm v0.17.0, thus there is a need to update the mix.exs in exrm v0.19.6

**Comment by bitwalker** [comment]
Pushed a new release of exrm using 0.17 of conform.
